packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -215,6 +232,14 @@ class SKProductDiscountWrapper {
   /// and their units and duration do not have to be matched.
   final SKProductSubscriptionPeriodWrapper subscriptionPeriod;
 
+  /// A string used to uniquely identify a discount offer for a product.
```

Comment:
It would be good to mention the default value here. I am also wondering if it would't be better to allow the `identifier` field to be `null` so it matches the native implementation. @stuartmorgan how do you feel about this?

Reply:
Done.